static/app/views/ddm/chart.tsx

```diff
@@ -51,6 +59,9 @@ export const MetricChart = forwardRef<ReactEchartsRef, ChartProps>(
       focusArea,
       removeFocusArea,
       height,
+      correlations,
+      onSampleClick,
+      highlightedSampleId,
```

Comment:
At some point we could / should split this component into a `BaseMetricsChart` and a `ExplorerChart` which adds the ddm page exclusive features.